### PR TITLE
Suomi text updates

### LINKED ISSUE #2574: Update text for Suomi NPP
Update layer description text to ensure Worldview states "Suomi NPP" across the app.

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #2574 .

Standardized text across descriptions when describing Suomi NPP, Kelvin, Earth, etc.

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/layers/airs/AIRS_L2_Surface_Skin_Temperature_Night.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #### Description
-Temperature of the solid or liquid surface of Earth in units of kelvin. Skin temperature refers to the temperature of the surface layer of the Earth, in contrast to the air temperature above the ground. Unlike temperatures in the atmosphere at higher altitudes, surface skin temperature can experience large changes between day and night due to solar heating.
+Temperature of the solid or liquid surface of the Earth in units of  (K). Skin temperature refers to the temperature of the surface layer of the Earth, in contrast to the air temperature above the ground. Unlike temperatures in the atmosphere at higher altitudes, surface skin temperature can experience large changes between day and night due to solar heating.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/airs/AIRS_L2_Surface_Skin_Temperature_Night.md]:
```suggestion
Temperature of the solid or liquid surface of the Earth in units of Kelvin (K). Skin temperature refers to the temperature of the surface layer of the Earth, in contrast to the air temperature above the ground. Unlike temperatures in the atmosphere at higher altitudes, surface skin temperature can experience large changes between day and night due to solar heating.
```

Comment (minniewong) [reply config/default/common/config/metadata/layers/airs/AIRS_L2_Surface_Skin_Temperature_Night.md]: Thanks for catching that!

File: config/default/common/config/metadata/layers/viirs/CorrectedReflectance.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ False Color: Red = M3, Green = I3, Blue = M11
...
 
-The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi-National Polar orbiting Partnership (S-NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS). The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
+The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi National Polar orbiting Partnership (S-NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS). The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/viirs/CorrectedReflectance.md]:
```suggestion
The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi National Polar orbiting Partnership (Suomi NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS). The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
```

File: config/default/common/config/metadata/layers/viirs/CorrectedReflectance.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ This combination is most useful for distinguishing burn scars from naturally low
...
 
-The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi-National Polar orbiting Partnership (S-NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS).The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
+The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi National Polar orbiting Partnership (S-NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS).The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/viirs/CorrectedReflectance.md]:
```suggestion
The Visible Infrared Imaging Radiometer Suite (VIIRS) Corrected Reflectance imagery is available only as near real-time imagery. The VIIRS instrument in on board the joint NASA/NOAA Suomi National Polar orbiting Partnership (Suomi NPP) satellite. The imagery can be visualized in Worldview and the Global Imagery Browse Services (GIBS).The sensor resolution is 750 m and 375 m (M Bands are 750 m, I Bands are 375 m), imagery resolution is 250 m, and the temporal resolution is daily.
```

Comment (minniewong) [reply config/default/common/config/metadata/layers/viirs/CorrectedReflectance.md]: thanks again!

**Pull request merged**